### PR TITLE
Simplify run gradle tasks

### PR DESCRIPTION
**Pull request by dima-avdeev-jb**
Maybe now we can simplify intermediate gradle tasks?

### CODE REVIEW

**Comment by Walingar** [comment]
Hm, not sure about this approach. We need to provide IDEA configuration then also, since for example from my side I don't use these scripts, but use gradle configuration

**Comment by dima-avdeev-jb** [comment]
IDEA run configurations in `.run/desktop` directory depends on the same tasks, like here:
(File test.run.xml)

image

**Comment by Walingar** [comment]
@dima-avdeev-jb it means that we will have two places with the same (and already not the same) configurations

**Comment by dima-avdeev-jb** [comment]
Yeah, we already have it.
For test tasks, we can stay it as is and simplify gradle configuration.

**Comment by Walingar** [comment]
So, I would suggest having a "composite" gradle task and using it in Development.md and IDEA run configuration

File: mpp/build.gradle.kts [hunk 1]
```diff
@@ -130,9 +130,9 @@ tasks.register("testWeb") {
     dependsOn(":compose:runtime:runtime:jsTest")
 }
 
-tasks.register("testAll") {
-    dependsOn(":mpp:testComposeJbDesktop")
-    dependsOn(":mpp:testComposeJbWeb")
+tasks.register("testAll") { //todo maybe deprecated?
```
Comment (Walingar) [inline mpp/build.gradle.kts]: Yeah, noticed that this is not really needed, let's remove it

Comment (dima-avdeev-jb) [reply mpp/build.gradle.kts]: Removed

**Review by Walingar — APPROVED** [review]

**Review by eymar — APPROVED** [review]

**Pull request merged**